src/openrct2/ride/Vehicle.cpp

```diff
@@ -763,73 +763,73 @@ static const struct
 
 // clang-format on
 
-static bool vehicle_move_info_valid(int32_t cd, int32_t typeAndDirection, int32_t offset)
+static bool vehicle_move_info_valid(int32_t trackSubposition, int32_t typeAndDirection, int32_t offset)
 {
-    if (cd >= static_cast<int32_t>(std::size(gTrackVehicleInfo)))
+    if (trackSubposition >= static_cast<int32_t>(std::size(gTrackVehicleInfo)))
     {
         return false;
     }
     int32_t size = 0;
-    switch (cd)
+    switch (trackSubposition)
     {
-        case 0:
+        case VEHICLE_TRACK_SUBPOSITION_0:
             size = 1024;
             break;
-        case 1:
+        case VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_1:
             size = 692;
             break;
-        case 2:
-        case 3:
-        case 4:
+        case VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_2:
+        case VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_3:
+        case VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_4:
             size = 404;
             break;
-        case 5:
-        case 6:
-        case 7:
-        case 8:
+        case VEHICLE_TRACK_SUBPOSITION_GO_KART_5:
+        case VEHICLE_TRACK_SUBPOSITION_GO_KART_6:
+        case VEHICLE_TRACK_SUBPOSITION_GO_KART_7:
+        case VEHICLE_TRACK_SUBPOSITION_GO_KART_8:
             size = 208;
             break;
-        case 9:
-        case 10:
-        case 11:
-        case 12:
-        case 13:
-        case 14:
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_A_9: // VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_START_9
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_A_10:
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_B_11:
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_B_12:
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_PATH_C_13:
+        case VEHICLE_TRACK_SUBPOSITION_MINI_GOLF_BALL_PATH_C_14:
             size = 824;
             break;
-        case 15:
-        case 16:
+        case VEHICLE_TRACK_SUBPOSITION_REVERSER_RC_15:
+        case VEHICLE_TRACK_SUBPOSITION_REVERSER_RC_16:
             size = 868;
             break;
     }
     if (typeAndDirection >= size)
     {
         return false;
     }
-    if (offset >= gTrackVehicleInfo[cd][typeAndDirection]->size)
+    if (offset >= gTrackVehicleInfo[trackSubposition][typeAndDirection]->size)
     {
         return false;
     }
     return true;
 }
 
-const rct_vehicle_info* vehicle_get_move_info(int32_t cd, int32_t typeAndDirection, int32_t offset)
+const rct_vehicle_info* vehicle_get_move_info(int32_t track_status, int32_t typeAndDirection, int32_t offset)
```

Comment:
Modify `vehicle_get_move_info` to change the variable names to `trackSubPosition`